integration-tests/kyuubi-trino-it/src/test/scala/org/apache/kyuubi/it/trino/server/TrinoFrontendSuite.scala

```diff
@@ -46,16 +67,8 @@ class TrinoFrontendSuite extends WithKyuubiServer with SparkMetadataTests {
   override protected val password: String = ""
 
   override def beforeAll(): Unit = {
-    super.beforeAll()
-
     // eagerly start spark engine before running test, it's a workaround for trino jdbc driver
     // since it does not support changing http connect timeout
-    try {
-      withJdbcStatement() { statement =>
-        statement.execute("SELECT 1")
-      }
-    } catch {
-      case NonFatal(e) =>
-    }
```

Comment:
please do not remove this workaround, see the comment ..

Reply:
ok，I thought it was for init testing .. it can skip the timeout exception